Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -305,6 +305,38 @@ mode would look as follows:
         --set global.k8sServiceHost=API_SERVER_IP \\
         --set global.k8sServicePort=API_SERVER_PORT
 
+NodePort XDP Acceleration
+*************************
+
+Cilium has built-in support for accelerating NodePort, ExternalIPs and LoadBalancer
+services for the case where the arriving request needs to be pushed back out of the
+node when the backend is located on a remote node. This acting as a hairpinned load
+balancer can be handled by Cilium at the XDP (eXpress Data Path) layer where BPF is
+operating directly in the networking driver instead of a higher layer.
+
+The mode setting ``global.nodePort.acceleration`` allows to enable this acceleration
+through the options ``native`` or ``generic``. The option ``none`` is the default
+and disables the acceleration. The setting of ``global.nodePort.acceleration=native``
+should **always** be preferred over ``generic`` since the latter is a fallback only
+suitable for testing drivers which do not have XDP support. The majority of drivers
+supporting 10G or higher rates also support native XDP on a recent kernel. For cloud
+based deployments most of these drivers have SR-IOV variants that support XDP as well.
```

Comment:
This paragraph spends about three sentences (the majority) encouraging `native` mode (which is good!), but it doesn't state when you might need `generic` mode.

Reply:
Makes sense, will elaborate there.